src/backupNg/backupNg.spec.js

```diff
@@ -377,4 +377,193 @@ describe("backupNg", () => {
     });
     expect(vmTask.data.id).toBe(vmId);
   });
+
+  test("execute three times a rolling snapshot, a delta backup and a CR, revert to an old state & restore the backups", async () => {
+    jest.setTimeout(8e4);
+    const vmId = await xo.createTempVm({
```

Comment:
The `vmId` should be configurable to let we test on a VM with big disks. A VM with tools will be needed for CR restoration.